packages/request-client.js/src/api/escrow.ts

```diff
@@ -0,0 +1,51 @@
+import { PaymentTypes, RequestLogicTypes } from '@requestnetwork/types';
+import { ICurrencyManager } from '@requestnetwork/currency';
+import { Erc20PaymentNetwork } from '@requestnetwork/payment-detection';
+import { AdvancedLogic } from '@requestnetwork/advanced-logic';
+
+/**
+ * Class representing an escrow.
+ * Instanced of this class can e
+ */
+
+export default class Escrow {
```

Comment:
I don't get this class. Why do we need another object than the request? The escrow data should be part of the extension data